humility-core/Cargo.toml

```diff
@@ -35,6 +35,7 @@ strum_macros = "0.24.1"
 clap = "3.0.12"
 roxmltree = "0.15"
 xmlparser = "0.13"
+hex = "0.4.3"
```

Comment:
You'll have to update the nix hash within `humility.nix` with this change.  The CI _should_ tell you what the new hash is.  I need to transition to [naersk](https://github.com/nix-community/naersk) at some point so manually changing the hash is not required.

Reply:
Hash has been updated in branch update.